generated/rflx-enumeration-generic_message.ads

```diff
@@ -203,6 +203,10 @@ is
        and Predecessor (Ctx, F_Priority) = Predecessor (Ctx, F_Priority)'Old
        and Valid_Next (Ctx, F_Priority) = Valid_Next (Ctx, F_Priority)'Old;
 
+   pragma Warnings (Off, "precondition is always False");
+
+   pragma Warnings (On, "precondition is always False");
+
```

Comment:
Is this intended?

Reply:
No, I will fix that.

Reply:
Fixed in a652916. `Set_X_Empty` procedures are only added now, if a field can possibly be empty. So a suppression of warnings is not needed anymore.